### ISSUE TITLE
Graphistry Embed Model Statistics [FEA]

### ISSUE THREAD

**Issue by molynyk** [graphistry/pygraphistry#433]
@silkspace  @tanmoyio 

**Is your feature request related to a problem? Please describe.**
I'm working on a Morpheus pipeline that trains and applies graphistry's RGCN model (g.embed()) to graph data and logs the model, parameters, and metrics in MLFlow for reference. Ideally, I'd like to be able to log to MLFlow metrics like the loss and accuracy per epoch.

**Describe the solution you'd like**
It would be nice if there is either an attribute in the graphistry plotter object or else a method in the plotter that will return these values.

i.e. something like: g.accuracy, g.loss, etc.

**Describe alternatives you've considered**
I've considered pulling out the RGCN model from the graphistry object and at least getting the final accuracy and loss via DGL/pytorch methods.

**Comment by silkspace** [comment]
Thanks @molynyk 

Future roadmap has Mlflow/Optuna decorators on all models -- all metrics would be exposed in mlflow UI. 

a quick fix could be `g._metrics` with keys `accuracy`, `loss`, etc

We will prioritize and update shortly.

**Comment by tanmoyio** [comment]
Thanks for raising the issue, Will it be helpful if you get the logs after the training gets finished or you want training time logging ?

**Comment by molynyk** [comment]
Either will work. I haven't toyed too much with logging metrics to a run as they're created, so I don't know how that works, but after the model's done training/the run's complete will definitely work.

And sounds good, @silkspace

**Comment by jrmendeshurb4** [comment]
ok

**Comment by lmeyerov** [comment]
@tanmoyio our charts have mlflow, can we try doing explicit mlflow support for whatever their API supports?
